flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/table/UniformShardAssignerFactory.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.flink.streaming.connectors.kinesis.table;
+
+import org.apache.flink.streaming.connectors.kinesis.KinesisShardAssigner;
+import org.apache.flink.streaming.connectors.kinesis.util.UniformShardAssigner;
+
+/**
+ * A {@link KinesisDynamicShardAssignerFactory} that loads the {@link UniformShardAssigner} shard
+ * assigner.
+ */
+public class UniformShardAssignerFactory implements KinesisDynamicShardAssignerFactory {
```

Comment:
Missing compatibility annotation, again suggest `@Internal` for this

Reply:
FYI - I followed the existing compatibility annotation in the underlying UniformShardAssigner (which is @PublicEvolving). Let me know if that's ok.